packages/ffe-cards-react/src/IconCard/IconCard.tsx

```diff
@@ -33,16 +38,17 @@ function IconCardWithForwardRef<As extends ElementType>(
             {...(rest as Record<string, unknown>)}
             ref={ref}
         >
-            {({ CardAction }) => (
-                <>
-                    {React.cloneElement(icon, {
+            {({ CardAction }) => {
+                const content = [
+                    React.cloneElement(icon, {
```

Comment:
Gikk det fint uten key i denne?

Reply:
Ja, faktisk. At den ene har key og den andre ikke er tydeligvis nok for linteren: den ser de har forskjellig key

Reply:
Ja, linteren tar det kanskej ikke men man får kanske noe logget i consolet når man kjører up det lokalt?